Configurations/50-nonstop.conf

```diff
@@ -0,0 +1,176 @@
+#### Nonstop configurations
+    "nonstop-common" => {
+        inherit_from     => [ "BASE_unix" ],
+        template         => 1,
+        cc               => "c99",
+        cflags           => add_before(picker(debug   => "-g -O0",
+                                              release => "-g -O2") ,"-Wextensions -Wnowarn=203,220,272,734,770,1506 -Wbuild_neutral_library"),
+        perl             => "/usr/bin/perl",
+        lflags           => "-lrld",
+        shared_target    => "self",
+        shared_cflag     => "",
+        shared_ldflag    => "-Wshared",
+        shared_extension => ".so",
+        enable           => ["egd"],
+        dso_scheme       => "DLFCN",
```

Comment:
If you add the following here, the macro `OPENSSL_SYS_TANDEM` will automatically be created for you:

``` perl
        sys_id           => 'TANDEM',
```

Also, every `OPENSSL_SYSNAME_TANDEM` in the C source should be renamed to `OPENSSL_SYS_TANDEM`, and you can remove `"OPENSSL_SYSNAME_TANDEM"` from every `defines` in this file.

Furthermoore, it might be a good idea to collect all the common `defines` values in one here, and please use `add()`, like this (here):

``` perl
        defines    => add("OPENSSL_VPROC=\$(OPENSSL_VPROC)", "_XOPEN_SOURCE", "_XOPEN_SOURCE_EXTENDED=1", "B_ENDIAN", "OPENSSL_SYSNAME_TANDEM"),
```

Reply:
This is a pretty big change. I like it a lot, however. It should not be a problem to do. Is there a deadline on this PR?

Reply:
Hmmm, there is, actually, more or less tomorrow . I could simply approve as is, and we have another PR to streamline it.

Reply:
My understanding is that in order to make 3.0 it needs to be in an approved state by today, basically.

Reply:
> 
> 
> Hmmm, there is, actually, more or less tomorrow . I could simply approve as is, and we have another PR to streamline it.

I will commit to that streamlining (pun intended).